src/ConstrainedPhotometryModel.cc

```diff
@@ -251,10 +247,11 @@ std::shared_ptr<afw::image::PhotoCalib> ConstrainedFluxModel::toPhotoCalib(CcdIm
     // Now stitch them all together.
     auto transform = pixToFocal->then(chebyTransform)->then(zoomTransform);
     // NOTE: TransformBoundedField does not yet implement mean(), so we have to compute it here.
-    double mean = mapping->getChipMapping()->getParameters()[0] * visitTransfo->mean();
+    // TODO: restore this calculation as part of DM-16305
+    // double mean = mapping->getChipMapping()->getParameters()[0] * visitTransfo->mean(ccdBBoxInFocal);
```

Comment:
Why replace this with `oldPhotoCalib->getCalibrationMean()`?

(Also, the diff would be a lot easier to understand if you kept `mean` and just changed the definition.)

Reply:
Because that's the temporary fix I'm implementing now: the existing `integrate()` code sometimes produces a negative mean(), because it's integrating on too large of a bbox. DM-16305 is to implement a better integrator, but it was taking too long, so we went with a temporary fix.